spec/controllers/application_controller_spec.rb

```diff
@@ -33,7 +33,8 @@ def create
       WebMock.stub_request(:get, /\w/).
         to_raise(Excon::Errors::Timeout.new('Request Timeout'))
       post :create
-      expect(Instrumentation.custom_log_items[:api_request_count]).to eq(1)
+      # 2 because we now catch the retries
+      expect(Instrumentation.custom_log_items[:api_request_count]).to eq(2)
```

Comment:
i think the instrumentation payload contains a flag saying if it is a retry so this could actually be done